packages/environment-check/src/cli/index.ts

```diff
@@ -38,8 +38,9 @@ function getOptions(cliArgs: minimist.ParsedArgs): CheckEnvironmentOptions | und
     if (cliArgs._.length > 0) {
         options.workspaceRoots = cliArgs._;
     }
-    if (cliArgs.destination) {
-        options.destinations = Array.isArray(cliArgs.destination) ? cliArgs.destination : [cliArgs.destination];
+
+    if (cliArgs['destination']) {
```

Comment:
While the README is updated (https://github.com/SAP/open-ux-tools/pull/802/files#diff-3bb333279c9541e0e673c2c26086af636744d4200cbb1ce3d7205112bea1adc1R66) 
the help text for `--desination` is not updated yet:

```
$ bin/envcheck --help           

Usage: envcheck [<OPTIONS>] [<WORKSPACE_ROOT_A>] [<WORKSPACE_ROOT_..>]

Following <OPTIONS> are available:
    --destination <DESTINATION>         destination to perform deep check, multiple destionations can be passed
    --output json | markdown | verbose | zip | userDownload  format for output, if not specified all messages except 'info' are shown

<WORKSPACE_ROOT*>                       path the root folder of a workspace. Multiple roots can be defined.
                                        We search for apps with destinations in workspaces
```